utils/support.ts

```diff
@@ -46,12 +46,15 @@ export const somePercentFromTX = (api: ApiPromise, price: number | string) => {
   return asBalanceTransfer(api, resolveSupportAddress(api), fee)
 }
 
+export const royaltyFee = (price: string | number, royaltyPercent: number) =>
+  (Number(price) * royaltyPercent) / 100
```

Comment:
looks different as before, have you check if it doesn't break rmrk royalty?

Reply:
![83qxa3](https://github.com/kodadot/nft-gallery/assets/9987732/5d547460-76d6-45f8-a2cb-28c5e609baf6)